test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.junit.Before;
+
+public class UtilsTest {
+
+	public void testIsOneObjectNull() {
+		assertTrue(Utils.isAnyNull(null, "Test", "Test2"));
+	}
+
+	public void testMoreThanOneObjectNull() {
```

Comment:
Note the naming convention for test methods (the first part is feature/method under test), e.g. `isAnyNull_emptyList_falseReturned`